src/sentry/utils/outcomes.py

```diff
@@ -104,6 +139,8 @@ def track_outcome(org_id, project_id, key_id, outcome, reason=None, timestamp=No
                 }
             ),
         )
+        # outcomes originating in Sentry (sent by this function)
+        mark_outcome_signal_sent(project_id=project_id, event_id=event_id)
```

Comment:
Please move this to where the signal is emitted instead of the outcome. Since this is only required for signals, it will be more clear if it is located there. Also, once we stop emitting signals in Sentry (by removing the code), this will automatically be gone with it.